cranelift-wasm/src/environ/dummy.rs

```diff
@@ -152,6 +165,25 @@ impl DummyEnvironment {
     pub fn get_num_func_imports(&self) -> usize {
         self.info.imported_funcs.len()
     }
+
+    /// Return the name of the module within this `DummyEnvironment`, if any.
+    pub fn get_module_name(&self) -> Option<&str> {
+        self.module_name.as_ref().map(String::as_ref)
+    }
+
+    /// Return the name of the function, if a name for the function with
+    /// the corresponding index exists.
+    pub fn get_func_name(&self, func_index: FuncIndex) -> Option<&str> {
+        self.function_names.get(&func_index).map(String::as_ref)
+    }
+
+    /// Return the name of the local variable, if a name for the local with
+    /// the corresponding indices exists.
+    pub fn get_local_name(&self, func_index: FuncIndex, local_index: LocalIndex) -> Option<&str> {
+        self.local_names
+            .get(&(func_index, local_index))
+            .map(String::as_ref)
```

Comment:
Tried `.get(..).as_ref()` in these methods and was running into type errors bc of `Option<&String>` rather than `Option<&str>`. Not sure if I'm forgetting a better way to do this, so I figured these `String::as_ref` calls were worth pointing out :slightly_smiling_face: